spswarehouse/credentials.py.template

```diff
@@ -42,4 +42,16 @@ calpads_config = {
     "username": "",
     "password": "",
     "host": "https://www.calpads.org"
-}
\ No newline at end of file
+}
+
+canvas_config = {
```

Comment:
If we hope that other organizations might one day use `spswarehouse`, it could be useful to have a comment explaining where to find/create these credentials in Canvas.